webpack.config.js

```diff
@@ -74,4 +76,29 @@ const config = {
     ]
 }
 
-module.exports = config;
+// in production mode, webpack decorates class constructor names.
+// changing constructor names causes an issue for the following logic:
+//    if type.constructor.name !== 'AbortSignal'...
+// this issue is present in the package node-fetch - version "2.6.0", which is
+// used by the package @azure/ms-rest-js - version "2.6.0".
```

Comment:
>@azure/ms-rest-js - version "2.6.0" [](http://example.com/codeflow?start=23&length=35)

is this a workaround only for ms-rest-js 2.6.0 version? Is there a newer version of ms-rest-js that we can snap to to avoid this workaround?

Reply:
There is not - 2.6.0 is the latest :(